### PR TITLE
fix(EFB): Agnostic opt in to error reporting

### LINKED ISSUE #9232: FlyPad Error Reporting Toggle - Wrong Aircraft
### Aircraft Version

Development

### Build info

```json
"built": "2024-10-31T15:54:41+00:00",
    "ref": "refs/tags/v0.12.0",
    "sha": "2fe89113291f2478347dde0dc3228b55dc924f74",
    "actor": "FoxtrotSierra6829",
    "event_name": "manual",
    "pretty_release_name": "stable/v0.12.0",
    "version": "a380x-v0.12.0-rel.2fe8911"
```

### Describe the bug

FlyPad still shows A32NX when enabling error reporting
Screenshot 2024-11-01 073957

### Expected behavior

Should show A380

### Steps to reproduce

1. Open FlyPad 
2. Settings
3. ATSU/AOC
4. Error Reporting - Toggle On

### References (optional)

_No response_

### Additional info (optional)

_No response_

### Discord Username (optional)

Yahtzee94

### PR DESCRIPTION
**Pull request by sschiphorst**
Fixes #9232

## Summary of Changes

Change error reporting popup to be plane agnostic rather than just A32NX

## Additional context

Discord username (if different from GitHub): Yahtzee94

## Testing instructions

1. Launch either aircraft at stand or runway
2. Power on EFB
3. In settings go to ATSU/AOC
4. Toggle errror reporting on
5. Confirm correct message appears

## How to download the PR for QA

Every new commit to this PR will cause new A32NX and A380X artifacts to be created, built, and uploaded.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the left side, find and click on the **PR Build** tab
1. Click on either **flybywire-aircraft-a320-neo** or **flybywire-aircraft-a380-842** download link at the bottom of the page

### CODE REVIEW

**Review by 2hwk — APPROVED** [review]

File: fbw-common/src/systems/shared/src/FbwAircraftSentryClient.ts [hunk 1]
```diff
@@ -161,8 +161,8 @@ export class FbwAircraftSentryClient {
     const popup = new PopUpDialog();
 
     return new Promise<boolean>((resolve) => {
-      popup.showPopUp(
-        'A32NX - ERROR REPORTING',
+      popup.showPopUp( /
```
Comment (2hwk) [inline fbw-common/src/systems/shared/src/FbwAircraftSentryClient.ts:164]:
```suggestion
      popup.showPopUp(
```

The `/` character is causing the build to fail. Remove to correct the syntax please.

**Review by 2hwk — CHANGES_REQUESTED** [review]

**Review by sschiphorst — COMMENTED** [review]
Have amended

**Review by tracernz — APPROVED** [review]

**Review by 2hwk — APPROVED** [review]

File: fbw-common/src/systems/shared/src/FbwAircraftSentryClient.ts [hunk 2]
```diff
@@ -161,8 +161,8 @@ export class FbwAircraftSentryClient {
     const popup = new PopUpDialog();
 
     return new Promise<boolean>((resolve) => {
-      popup.showPopUp(
-        'A32NX - ERROR REPORTING',
+      popup.showPopUp(
```
Comment (tracernz) [inline fbw-common/src/systems/shared/src/FbwAircraftSentryClient.ts:164]: You've added a trailing space here. The lint won't like that.

**Review by 2hwk — APPROVED** [review]

**Comment by 2hwk** [comment]
Quality Assurance Tester Report

Discord Username: 2cas
Object of testing: #9257
Aircraft:  A380X
Tier of Testing: 1
Date: 15/11/2024 

Testing Process: 
- [x] Launch either aircraft at stand or runway
- [x] Power on EFB
- [x] In settings go to ATSU/AOC
- [x] Toggle errror reporting on
- [x] Confirm correct message appears

Testing Results: 
QA Passed

Negatives:       
- N/A

Conclusions: 
- LGTM

**Pull request merged**